### PR TITLE
views: update import path

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/inveniosoftware/invenio-rdm-records/issues/1479

### CODE REVIEW

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html [hunk 1]
```diff
@@ -9,11 +9,15 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{%- macro preview_file(preview_endpoint, pid_value, filename, is_preview, id='preview-iframe', width='100%', height='400' ) %}
+{%- macro preview_file(preview_endpoint, pid_value, filename, is_preview, include_deleted, id='preview-iframe', width='100%', height='400' ) %}
```
Comment (kpsherva) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html:12]: could you check if we need to update templates for zenodo and cds?

File: invenio_app_rdm/records_ui/views/decorators.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Routes for record-related pages provided by Invenio-App-RDM."""
-
+from copy import deepcopy
```
Comment (kpsherva) [inline invenio_app_rdm/records_ui/views/decorators.py:11]: not needed anymore?

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html (hunk 1 above)
Comment (jrcastro2) [reply invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html:12]: yes, here the PRs:
- https://github.com/zenodo/zenodo-rdm/pull/534
- https://github.com/CERNDocumentServer/cds-rdm/pull/106

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html [hunk 3]
```diff
@@ -240,9 +240,9 @@ <h2 id="files-heading">{{ _('Files') }}</h2>
 
                     {%- if files|has_previewable_files -%}
                       {%-set preview_file = files|select_preview_file(default_preview=record.files.default_preview) %}
-                      {{ preview_file_box(preview_file, record.id, is_preview, record) }}
+                      {{ preview_file_box(preview_file, record.id, is_preview, record, include_deleted) }}
```
Comment (kpsherva) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html:243]: is it still needed?

**Pull request merged**